SeaPublicWebsite/Models/Feedback/FeedbackFormViewModel.cs

```diff
@@ -0,0 +1,12 @@
+﻿using GovUkDesignSystem.Attributes.ValidationAttributes;
+
+namespace SeaPublicWebsite.Models.Feedback;
+
+public class FeedbackFormViewModel
+{
+    [GovUkValidateRequired(ErrorMessageIfMissing = "Enter what your were doing")]
+    public string WhatUserWasDoing { get; set; }
+    
+    [GovUkValidateRequired(ErrorMessageIfMissing = "Enter what you would like to tell us")]
+    public string WhatUserToldUs { get; set; }
+}
```

Comment:
These are the fields of the form. We expect they will change with time, for now I named them to resemble the question asked.